architecture/additional_concepts/authorization.adoc

```diff
@@ -880,6 +880,43 @@ in their SCC set.  This allows cluster administrators to run pods as any
 user by without specifying a `RunAsUser` on the pod's `SecurityContext`.  The
 administrator may still specify a `RunAsUser` if they wish.
 
+[[rbac-to-scc]]
+==== Role-Based Access to SCCs ====
+Starting with {product-title} 3.11, you can specify Security Context Constrains as a
+resource that is handled by RBAC. This allows you to
+scope the access to your SCCs to a certain project or cluster wide. Assigning users/groups
```

Comment:
I'd s/cluster wide/to the entire cluster

s/"users/groups"/users or groups

What's the default scope of the SCC?

Reply:
By default, you specify users/groups in the appropriate SCC. That is therefore set to the whole cluster since SCCs are defined cluster-wide.